packages/node_modules/@webex/internal-plugin-wdm/src/device/device.js

```diff
@@ -52,6 +52,7 @@ const Device = WebexPlugin.extend({
     helpUrl: 'string',
     intranetInactivityDuration: 'number',
     intranetInactivityCheckUrl: 'string',
+    inNetworkInactivityDuration: 'number', // Newly added duration for internal network inactivity
```

Comment:
Will there be different values for in network and out of network timers? In other words, will there still be two values in control hub or just one?

Reply:
Yes there are 2 different values for the timers and control hub has a UI that allows admins to choose them.